fluxfw-gcp/tf/examples/README.md

```diff
@@ -0,0 +1,23 @@
+# fluxfw-gcp Examples
+
+This directory contains a collection of examples of deploying and using the flux-framework on GCP.
+Each of the examples deploy on a _foundation_ of network, iam, and storage components defined in the
+[foundation]() directory.
+
+## Usage
+
+Before you deploy an example the foundation they depend on must be deployed. The documentation in 
+the [foundation README]() describes how to deploy the foundation.
+
+For the impatient the simplest foundation deployment is realized with the command:
+
+```bash
+cd foundation
+make
+```
+
+Once the foundation is deployed you can visit any of the example directories and follow the steps
+in the their README documentation to deploy the example.
+
+_Note_ the foundation only needs to be deployed once. Any of the examples, or a collection of them,
+can be deployed simultaneously on the foundation.
```

Comment:
I recommend providing very literal instructions for someone who won't actually _read_ this, but who might be clicking through.  I.e., subsequent commands that either assume they're still in the `foundation` dir or renavigate from the toplevel.

Also, how does this behave in the cloudshell tutorial mode?  That's sort of the ultimate test of someone not reading but clicking through the steps sequentially.